gama/src/app/app.component.html

```diff
@@ -1,5 +1,3 @@
-<app-header></app-header>
```

Comment:
Se não for utilizar os 2 componentes que criei, o app-header e o app-footer é melhor excluir eles

Reply:
deve-se remover os arquivos e as importações desses componentes se não for mais utiliza